udata/harvest/backends/dcat.py

```diff
@@ -87,16 +94,18 @@ def parse_graph(self, url, fmt):
 
         return graph
 
+    def get_node_from_item(self, item):
+        if 'nid' in item.kwargs and 'type' in item.kwargs:
+            nid = item.kwargs['nid']
+            return URIRef(nid) if item.kwargs['type'] == 'uriref' else BNode(nid)
+
     def process(self, item):
         graph = Graph(namespace_manager=namespace_manager)
-        data = item.kwargs.get('graph', self.job.data['graph'])  # handles legacy graphs
```

Comment:
We don't handle `item.kwargs.get('graph')`? It was legacy?

Reply:
Yup, I _think_ it's related to https://github.com/opendatateam/udata/pull/2305